button/mega.js

```diff
@@ -0,0 +1,52 @@
+import { html } from '../template.js';
+import { button } from './base.js';
+ 
+export const mega = html`
+    ${button}
+    <style>
+        .btn.mega {
+            height: 48px;
+            min-width: 150px;
+            display: inline-block;
+
+            line-height: 48px;
+            outline: none;
+            overflow: hidden;
+            padding: 0 22px;
+            white-space: nowrap;
+            cursor: pointer;
+
+            background: rgba(255, 255, 255, 1);
+            color: var(--color-abbey);
+
+            border-color: var(--color-stone);
+            border-style: solid;
+            border-width: 1px;
+            border-radius: 5px;
+
+            font-size: 16px;
+            font-family: var(--font-body);
+            font-weight: bold;
+
+            text-align: center;
+            text-transform: uppercase;
```

Comment:
Can you check the properties already in buttons and remove the duplicates. It would make a smaller stylesheet for the mega and soft buttons